javascript/modules/client/test/integration/client.test.ts

```diff
@@ -420,6 +427,98 @@ describe("Client", () => {
       expect(typeof withdrawAction).toBe("object");
       expect(withdrawAction.data).toBeInstanceOf(Uint8Array);
     });
+    it("Should create a client and generate a grant action", () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+      const paramsArray: IPermissionParams[] = [
+        {
+          who: "0x0987654321098765432109876543210987654321",
+          where: "0x0987654321098765432109876543210987654321",
+          permission: Permissions.EXECUTE_PERMISSION,
+        },
+        {
+          who: "0x1234567890123456789012345678901234567890",
+          where: "0x1234567890123456789012345678901234567890",
+          permission: Permissions.UPGRADE_PERMISSION,
+        },
+      ];
+      let actions: DaoAction[] = [];
+      for (let i = 0; i < paramsArray.length; i++) {
+        const params = paramsArray[i];
+        actions.push(client.encoding.grantAction(daoAddress, params));
```

Comment:
Let's make sure that `daoAddress` doesn't have the same address `0x1234...` as many other test variables